app/src/main/java/com/cornellappdev/android/eatery/model/EateryModel.java

```diff
@@ -0,0 +1,234 @@
+package com.cornellappdev.android.eatery.model;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import com.google.android.gms.maps.model.LatLng;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.threeten.bp.LocalDateTime;
+import org.threeten.bp.ZonedDateTime;
+
+/**
+ * Created by JC on 2/15/18. This represents a single Cafeteria (either a cafe or a dining hall)
+ */
+
+public abstract class EateryModel implements Model, Cloneable, Serializable,
+    Comparable<EateryModel> {
+
```

Comment:
Here I think a newline is *justified*, so we can keep it

Reply:
This is because the class definition hit >100 characters, which per Google's style guide needs a new line. I personally don't agree with that.